opts/opts.go

```diff
@@ -177,7 +177,7 @@ type ValidatorFctType func(val string) (string, error)
 type ValidatorFctListType func(val string) ([]string, error)
 
 func ValidateLogOpts(val string) (string, error) {
-	allowedKeys := map[string]string{}
+	allowedKeys := map[string]string{"max-file": "1", "max-size": "2"}
```

Comment:
hmm, I really don't like this.
Could we just instantiate a logger and call a ValidateOpts (after adding this to the interface) on it?
This is kind of a weird one since the values don't get used right away.

Anyone else want to weigh in?